middleware.js

```diff
@@ -122,6 +122,7 @@ function publishStats(action, statsResult, eventStream, log) {
     modules: true,
     timings: true,
     hash: true,
+    errors: true,
```

Comment:
We'll need warnings too
```suggestion
    errors: true,
    warnings: true,
```